src/scripts/operations.js

```diff
@@ -171,8 +171,40 @@ const operations = {
         }
       }, { strict: false });
     }
-  }
+  },
+
+  'add-reviewed-field-to-images': {
+    getIds: async () => (
+      await Image.find().select('_id')
+    ),
+    update: async () => {
+      console.log('Adding reviewed field to all images...');
+
+      let skip = 0;
+      let limit = 1; // how many images to fetch at a time
```

Comment:
Keeping this at 1 for now just for testing, but potentially be doing batches of 1k-10k

Reply:
@jue-henry we should try to clock this on dev at different batch settings before trying on prod. I worry it might take a longggg time on prod.